aiops/opni-opensearch-update-service/opensearch-update-service/app/main.py

```diff
@@ -21,6 +24,8 @@
     "ctx._source.anomaly_predicted_count += 1; ctx._source.opnilog_anomaly = true;"
 )
 
+protobuf_conversion_dict = {"Id": "_id" ,"clusterId" : "cluster_id", "maskedLog": "masked_log", "anomalyLevel": "anomaly_level", "logType": "log_type","drainPretrainedTemplateMatched": "drain_pretrained_template_matched","inferenceModel": "inference_model","opnilogConfidence": "opnilog_confidence"}
```

Comment:
Why not rename the fields in the message to match the names you want?

Reply:
@tybalex, you had looked into this right on your end?

Reply:
bc rename fields will likely results in changes at some other places, such as UI  and potentially index templates. 
the name conversion here is really just between camel style and python style.